src/App.js

```diff
@@ -1,25 +1,57 @@
-import logo from './logo.svg';
-import './App.css';
+import foods from "./foods.json";
+import { Card} from "antd"; 
+import { Foodbox } from "./components/Foodbox";
+import { AddFoodForm } from "./components/AddFoodForm";
+import { useState } from "react";
+import { Search } from "./components/Search";
+
+
 
 function App() {
+  
+  // console.log(menu)
+  const [oldMenu, coolMenu] = useState(foods)
```

Comment:
const [menu, SetMenu] = useState(foods)
  
  is better had general words in developer context s`pecially with this states